src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final String[] colors = new String[] {"black", "white", "green", "red", "gray",
+            "purple", "pink", "yellow", "brown", "blue", "orange"};
+
+    public String randomColor() {
```

Comment:
method names should be verbs
```suggestion
    public String getRandomColor() {
```